shared/src/Datadog.Trace.ClrProfiler.Native/dynamic_dispatcher.cpp

```diff
@@ -128,9 +124,9 @@ namespace datadog::shared::nativeloader
                     {
                         // Convert possible relative paths to absolute paths using the configuration file folder as base
                         // (current_path)
-                        std::string absoluteFilepathValue = fs::absolute(filepathValue).string();
+                        std::string absoluteFilepathValue = (configFolder / filepathValue).string();
```

Comment:
I assume this uses the correct separator for the platform right? win/linux

Reply:
yes